lib/embed.dart

```diff
@@ -761,7 +810,8 @@ class Embed {
     }
     tabController.setTabVisibility(
         'test', context.testMethod.isNotEmpty && _showTestCode);
-    menuButton.toggleAttr('hidden', context.testMethod.isEmpty);
+    // menuButton.toggleAttr('hidden', context.testMethod.isEmpty);
```

Comment:
Is this left over from debugging something? Otherwise, please delete comment.

Reply:
Oops - I'll remove this line. The menu used to hide if there wasn't a reason to show it, so now that the null safety option is there it means that we should always keep it visible.